src/ameba/cli/cmd.cr

```diff
@@ -77,6 +77,8 @@ module Ameba::Cli
 
       parser.on("-c", "--config PATH",
         "Specify a configuration file") do |path|
+        raise ArgumentError.new("Unable to find config file #{path}") if !File.exists?(path) && !opts.skip_reading_config?
+
```

Comment:
Hey and thanks for your contribution. 

can we move this logic to the config itself? 

https://github.com/crystal-ameba/ameba/blob/8c9d234d0b06d945d0cf841dfa0596734acc715c/src/ameba/config.cr#L109-L128